dashmips/__main__.py

```diff
@@ -149,3 +149,7 @@ def main() -> NoReturn:
         # If a user encounters a traceback that should be a fatal issue
         print("dashmips encountered an error!", file=sys.stderr)
         print(ex, file=sys.stderr)
+    except KeyboardInterrupt as ex:
+        # Program should be closed
+        print("Program terminated.")
```

Comment:
We should do a run over all the python we've changed uses the logger instead of printing and make sure the logger prints to stderr when enabled
For now lets make this `print(..., file=sys.stderr)` use stderr.